api/tests/routes/native/v1/account_test.py

```diff
@@ -438,6 +438,92 @@ def test_account_creation_performance_tests(self, mocked_check_recaptcha_token_i
             == f"performance-tests_email-validation_{user.id}"
         )
 
+    @patch("pcapi.connectors.api_recaptcha.check_recaptcha_token_is_valid")
+    def test_should_not_save_trusted_device_when_no_info_provided(self, mocked_check_recaptcha_token_is_valid, client):
+        data = {
+            "email": "John.doe@example.com",
+            "password": "Aazflrifaoi6@",
+            "birthdate": "1960-12-31",
+            "notifications": True,
+            "token": "gnagna",
+            "marketingEmailSubscription": True,
+        }
+
+        client.post("/native/v1/account", json=data)
+
+        assert users_models.TrustedDevice.query.first() is None
+
+    @patch("pcapi.connectors.api_recaptcha.check_recaptcha_token_is_valid")
+    def test_should_not_save_trusted_device_when_no_device_id_provided(
+        self, mocked_check_recaptcha_token_is_valid, client
+    ):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Est ce que dans ce cas on ne voudrais pas aussi logguer une erreur ?

Reply:
On peut effectivement, si on reçoit un objet sans rien à l'intérieur ça vaut le coup